src/Sulu/Bundle/AdminBundle/Resources/js/services/Router/tests/Router.test.js

```diff
@@ -870,6 +1044,106 @@ test('Binding should not be updated if only data type changes', () => {
     expect(page.set.mock.calls).not.toContainEqual(['2']);
 });
 
+test('Binding should not be updated if the same object is set again', () => {
+    routeRegistry.getAll.mockReturnValue({
+        list: {
+            name: 'list',
+            type: 'list',
+            path: '/list',
+            attributeDefaults: {},
+        },
+    });
+
+    const filter = jest.fn(() => ({
```

Comment:
it took me some time to get my head around whats happening here. but cant provide a simpler solution 🙈